storage/shard.go

```diff
@@ -349,24 +351,13 @@ func (s *dbShard) Bootstrap(bs bootstrap.Bootstrap, writeStart time.Time, cutove
 	s.Unlock()
 
 	multiErr := xerrors.NewMultiError()
-	sr, err := bs.Run(writeStart, s.shard)
-	if err != nil {
-		renamedErr := fmt.Errorf("error occurred bootstrapping shard %d from external sources: %v", s.shard, err)
-		err = xerrors.NewRenamedError(err, renamedErr)
+	for id, dbBlocks := range bootstrappedSeries {
+		series, _, completionFn := s.writableSeries(id)
```

Comment:
? Doesn't look like it:
![screen shot 2016-09-01 at 5 12 38 pm](https://cloud.githubusercontent.com/assets/339615/18184484/51067b72-7067-11e6-8b09-f7715a487f90.png)